### PR TITLE
[MLRun] Upgrade ingress resource to use v1 API version

### PR DESCRIPTION
**Pull request by ffais**
### Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove irrelevant fields.]

- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### Description

Migrated ingress manifests to use the networking.k8s.io/v1 API version, available since v1.19.

### CODE REVIEW

**Comment by ffais** [comment]
Hi @Hedingber, I changed the ingresses to use the new API version. Are there any problems merging this?

File: stable/mlrun/templates/api-ingress.yaml [hunk 1]
```diff
@@ -28,14 +25,15 @@ spec:
 {{- end }}
   rules:
   {{- range .Values.api.ingress.hosts }}
-    - host: {{ .host | quote }}
+    - host: {{ . | quote }}
       http:
         paths:
-        {{- range .paths }}
-          - path: {{ . }}
+          - pathType: ImplementationSpecific
```
Comment (alonmr) [inline stable/mlrun/templates/api-ingress.yaml]: please make the `pathType` overridable in the values file and keep the `paths` list to allow defining multiple paths as follows (mind the indentation):
```
      hosts:
        - host: chart-example.local
          paths:
            - pathType: ImplementationSpecific
              path: /
```

**Review by alonmr — CHANGES_REQUESTED** [review]
Hey @ffais, thanks for the contribution! 
Please make the following changes:
1. Bump the chart version to 0.9.15
2. Retain the `paths` list as stated below
3. Use `.Values.api` and `.Values.ui` instead of `.Values.dashboard` in the matching ingress templates.

File: stable/mlrun/templates/api-ingress.yaml (hunk 1 above)
Comment (ffais) [reply stable/mlrun/templates/api-ingress.yaml]: Hi @alonmr, I made the requested changes, in addition I also upgrade the api-chief ingress.

File: stable/mlrun/templates/api-ingress.yaml [hunk 2]
```diff
@@ -30,12 +26,14 @@ spec:
   {{- range .Values.api.ingress.hosts }}
     - host: {{ .host | quote }}
       http:
-        paths:
-        {{- range .paths }}
-          - path: {{ . }}
+          {{- range .paths }}
```
Comment (alonmr) [inline stable/mlrun/templates/api-ingress.yaml:30]: missing the `paths`
```suggestion
        paths:
        {{- range .paths }}
```

**Review by alonmr — CHANGES_REQUESTED** [review]
@ffais Thanks again! looks great. Small touch up needed.

File: stable/mlrun/templates/api-ingress.yaml (hunk 2 above)
Comment (ffais) [reply stable/mlrun/templates/api-ingress.yaml:30]: yep, my mistake :-)

**Review by alonmr — APPROVED** [review]

**Review by liranbg — APPROVED** [review]

**Pull request merged**